src/OpenTelemetry.Exporter.Jaeger/Implimentation/JaegerThriftClientTransport.cs

```diff
@@ -0,0 +1,108 @@
+// <copyright file="JaegerThriftClientTransport.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Jaeger.Implimentation
+{
+    using System;
+    using System.IO;
+    using System.Net.Sockets;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Thrift.Transports;
+
+    public class JaegerThriftClientTransport : TClientTransport
+    {
+        private readonly UdpClient udpClient;
+        private readonly MemoryStream byteStream;
+        private bool isDisposed = false;
+
+        public JaegerThriftClientTransport(string host, int port)
+        {
+            this.byteStream = new MemoryStream();
+            this.udpClient = new UdpClient();
+            this.udpClient.Connect(host, port);
+        }
+
+        public override bool IsOpen => this.udpClient.Client.Connected;
+
+        public override void Close()
```

Comment:
Consistent style with the property above:
```suggestion
        public override void Close() => this.udpClient.Close();
```
(Also it looks so short and nice IMO :)